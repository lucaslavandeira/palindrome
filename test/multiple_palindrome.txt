Somos los primeros en completar el TP 0.
Ojo que La fecha de entrega del TP0 es el martes 12 de septiembre.

