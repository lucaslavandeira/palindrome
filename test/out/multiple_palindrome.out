Somos
0
Ojo
